### DISCUSSION TITLE
Teaful js website?

### DISCUSSION THREAD

**Discussion by shubhamV123** [aralroca/fragstore#42] [Ideas]
Hi Everyone,

I came to know about teaful recently. Library looks awesome. I want to know about if you have a plan to make doc website.I am interested in giving contribution in that. I can make a draft website to show. Just bringing up if you haven't done anything on that I can start on that.

For doc site, I will most probably use [Docusaurus](https://docusaurus.io)

If you have anything on mind. Open for feedbacks!

**Comment by danielart** [comment]
Hi @shubhamV123 ! absolutely! is in our roadmap 🚀 , and we have a[ pull request here](https://github.com/teafuljs/teaful/pull/32), @siddharthborderwala is working on that, and we're gonna help him to accomplish. Your help is more than welcome! big thanks for your interest and your idea, we put some feedback about doc page on that pull request. Feel free to write your opinion and don't hesitate to purpose new things, we're growing fast as a community 💪

**Reply by shubhamV123** [reply]
Thanks @danielart Will look into that
